bb8/src/internals.rs

```diff
@@ -255,10 +275,16 @@ pub(crate) struct AtomicStatistics {
     pub(crate) get_direct: AtomicU64,
     pub(crate) get_waited: AtomicU64,
     pub(crate) get_timed_out: AtomicU64,
+    pub(crate) get_waited_time_micro: AtomicU64,
+    pub(crate) connections_created: AtomicU64,
+    pub(crate) connections_broken_closed: AtomicU64,
+    pub(crate) connections_invalid_closed: AtomicU64,
+    pub(crate) connections_max_lifetime_closed: AtomicU64,
+    pub(crate) connections_max_idle_timeout_closed: AtomicU64,
```

Comment:
Let's skip the `connections_` prefixes.

Reply:
For all of them? at least at first sight the `connections_created`, `connections_broken_closed`, `connections_broken_closed` and `connections_invalid_closed` without the `connections_` prefix will seem a bit off.

Also it follows the same practice for the `get_` Kind, where we prefixed it and IMO helps to the user to understand the scope of them, same would happen for `connections_`

Reply:
Okay, let's keep `connections_` but I'd like `closed` moved directly after `connections_` and rename `max_idle_timeout` to `idle_timeout`.